src/users/users.module.ts

```diff
@@ -0,0 +1,9 @@
+import { Module } from '@nestjs/common';
+import { UsersService } from './users.service';
+import { PrismaService } from '../database/prisma.service';
+import { MailService } from '../mail/mail.service';
+
+@Module({
+  providers: [UsersService, PrismaService, MailService],
```

Comment:
hmm, is this not returning an error? You provide `MailService` in `MailModule` already 🤔  I think you should import `MailModule` here.